[Zone Versioning] API Access prereq (#11870)
cloudflare/cloudflare-docs: content/version-management/_index.md

- Your zone has migrated to use [Custom Rules](/waf/reference/migration-guides/firewall-rules-to-custom-rules/) instead of Firewall Rules (deprecated).
- Your account uses the [new WAF](https://blog.cloudflare.com/new-cloudflare-waf/) (if not, contact your account team).
- Your user account must have an API Key provisioned (if not, [view your API Key](/fundamentals/api/get-started/keys/#view-your-global-api-key)).

> - Your user account must have API Access enabled (learn how to [control API Access](/fundamentals/api/how-to/control-api-access/)).

Reviewer: ```suggestion
- Your user account must have API Access enabled. Refer to [control API Access](/fundamentals/api/how-to/control-api-access/) for more information.
```
